ipatests/test_integration/test_epn.py

```diff
@@ -641,6 +642,7 @@ def test_EPN_nbdays_input_4(self):
     #  2) tweak the template
     #  3) add starttls
 
+    @skip_if_fips()
```

Comment:
I don't think that this skip is needed, the run from Nightly-STIG-Tier2/RHEL9.3/2023-09-15_00-42 shows a successful test.